docs/deb-package-install.adoc

```diff
@@ -1,6 +1,10 @@
 == Installing aktualizr via debian package
 
-Aktualizr makes .deb packages available via the https://github.com/advancedtelematic/aktualizr/releases[GitHub releases page].
+Aktualizr makes .deb packages available via the https://github.com/advancedtelematic/aktualizr/releases[GitHub releases page]. Download and install the .deb package, for example using dpkg:
+
+----
+sudo dpkg -i aktualizr.deb
```

Comment:
Might be better to recommend using apt instead; that way the dependencies will get installed automatically:

```
wget https://github.com/advancedtelematic/aktualizr/releases/download/2018.3/aktualizr.deb
sudo apt install ./aktualizr.deb
```